compiler/testData/diagnostics/tests/incompleteCode/diagnosticWithSyntaxError/valWithNoNameInBlock.fir.kt

```diff
@@ -0,0 +1,50 @@
+// !DIAGNOSTICS: -UNUSED_PARAMETER -UNUSED_VARIABLE
+
+fun println(x: String) {
+}
+
+fun run(block: () -> Unit) {}
+
+val propertyNameOnTheNextLine = 1
+
+fun foo() {
+    val<!SYNTAX!><!>
+    println("abc")
+
+    val<!SYNTAX!><!>
+    run {
+        println("abc")
+    }
+
+    val<!SYNTAX!><!>
+    if (1 == 1) {
+
+    }
+
+    val<!SYNTAX!><!>
+    (1 + 2)
+
+    // `propertyNameOnTheNextLine` parsed as simple name expression
+    val<!SYNTAX!><!>
+    propertyNameOnTheNextLine
+
+    val<!SYNTAX!><!>
+    // comment
+    propertyNameOnTheNextLine
+
+    val<!SYNTAX!><!> /* comment */
+    propertyNameOnTheNextLine
+
+    // Correct properties
+    val
+    property1 = 1
+
+    val
+    propertyWithBy by <!DELEGATE_SPECIAL_FUNCTION_MISSING!><!UNRESOLVED_REFERENCE!>lazy<!> { 1 }<!>
```

Comment:
Ideally we should skip reporting this since `lazy` is unresolved. But there does not seem to be a way to communicate this between `ErrorNodeDiagnosticCollectorComponent` and the standalone checker.